docs/install.md

```diff
@@ -103,7 +103,7 @@ For example, you can add the following to your `package.json` to have ESLint and
     }
   },
   "lint-staged": {
-    "**/*": ["eslint --fix", "prettier --write"]
+    "**/*": ["eslint --fix", "prettier --write --ignore-unknown"]
```

Comment:
```suggestion
    "*": "prettier --write --ignore-unknown"
```

I don't think we need mention eslint. @lydell ?

Reply:
It's important to show that it should be run before Prettier, not after.

Reply:
The stuff I added is copy-paste from @kachkaev, I believe. I don’t use lint-staged myself.

Reply:
@thorn0 

What if config like this

```
    "*.js": "eslint --fix"
    "*": "prettier --write --ignore-unknown"
```

😄

Reply:
Seems the key order matters

https://github.com/okonet/lint-staged/blob/86c9ed2bde1245a702635b42a57b92b809340b0e/lib/generateTasks.js#L31

Reply:
People also use ESLint for `*.jsx`, TypeScript, etc., so let's just write something like "If you use ESLint, make sure lint-staged runs it before Prettier, not after."

Reply:
Why does the order matter?

Reply:
Because ESLint changes code when it fixes things. Then Prettier formats it.